js/src/orange-navbar.js

```diff
@@ -0,0 +1,40 @@
+import EventHandler from './dom/event-handler'
```

Comment:
You will need ton review this file in order to follow the guidelines of Bootstrap and Boosed: 

/**
 * --------------------------------------------------------------------------
 * Boosted (v5.1.3): orange-navbar.js
 * Licensed under MIT (https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/LICENSE)
 * --------------------------------------------------------------------------
 */

<imports>

/**
 * Constants
 */

<constants>

/**
 * Class definition
 */

<...>

 // Getters

<...>

 // Public

<...>

 // Static

<...>

/**
 * Data API implementation
 */

<...>

/**
 * jQuery
 */

<...>